src/modules/dashboard/components/ExpenditurePage/TimeRelativeShort/TimeRelativeShort.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { HTMLAttributes } from 'react';
+import { FormattedRelativeTime } from 'react-intl';
+
+interface Props extends HTMLAttributes<HTMLSpanElement> {
+  /** Time in seconds to update */
+  updateInterval?: number;
+  /** Datetime to calculate relative/difference from. */
+  value: Date;
+  formatting?: 'short' | 'long' | 'narrow';
+}
+
+const nearestIntervalOf = (value: number, increment: number) =>
+  Math.round(value / increment) * increment;
+
+const displayName = 'TimeRelative';
+
+const TimeRelativeShort = ({
+  value: valueProp,
+  updateInterval = 15,
+  formatting = 'long',
+  ...rest
+}: Props) => {
+  const valueInSeconds = Math.floor(
+    nearestIntervalOf(
+      (new Date(valueProp).getTime() - new Date().getTime()) / 1000,
+      updateInterval,
+    ),
+  );
+
+  return (
+    <span {...rest}>
+      <FormattedRelativeTime
+        numeric="auto"
+        updateIntervalInSeconds={updateInterval}
+        value={valueInSeconds}
+        // eslint-disable-next-line react/style-prop-object
```

Comment:
I don't think that there is a need for the `eslint-disable-next-line` here. When i removed it i didn't have any warning,